src/webapp/pages/home/HomePage.tsx

```diff
@@ -263,3 +256,49 @@ function useRedirectOnSinglePrimaryNode(
 
     return { isActive: isActive, currentPage: currentPage };
 }
+
+type UseTrackAnalyticsOnLoadProps = {
+    compositionRoot: CompositionRoot;
+    userLandings: Maybe<LandingNode[]>;
+};
+
+function useTrackAnalyticsOnLoad(props: UseTrackAnalyticsOnLoadProps) {
+    const { compositionRoot, userLandings } = props;
+
+    React.useEffect(() => {
+        const initLandings = userLandings?.filter(landing => landing.executeOnInit);
+        const pageType = initLandings && initLandings?.length > 1 ? "userLandings" : "singleLanding";
+        if (userLandings && userLandings.length > 1 && pageType === "userLandings") {
+            trackUserLanding(compositionRoot);
+        }
+
+        if (initLandings && initLandings.length > 0 && pageType === "singleLanding") {
+            const cuPage = initLandings[0];
+            if (!cuPage) return;
+            trackSingleLanding(cuPage, compositionRoot);
+        }
+    }, [compositionRoot, userLandings]);
+}
+
+function buildViewOptionsFromLanding(landing: LandingNode) {
+    const type = landing.type === "root" ? "landing" : landing.type;
+    return {
+        title: `Homepage - ${landing.name.referenceValue}`,
+        location: `${window.location.hash.split("?")[0]}home-page-app/${type}/${landing.id}`,
+    };
+}
+
+function trackSingleLanding(landing: LandingNode, compositionRoot: CompositionRoot) {
+    const viewOptions = buildViewOptionsFromLanding(landing);
+    compositionRoot.analytics.sendPageView(viewOptions);
+    compositionRoot.matomo.trackView(viewOptions);
+}
+
+function trackUserLanding(compositionRoot: CompositionRoot) {
+    const viewOptions = {
+        location: `${window.location.hash.split("?")[0]}home-page-app/available-landings`,
```

Comment:
`window.location.hash.split("?")[0]` is now spread 4 times along the code. Not sure but maybe we should merge the use of it